app/models/concerns/pick_up_order.rb

```diff
@@ -0,0 +1,89 @@
+class PickUpOrder < ActiveRecord::Base
+  # rubocop:disable Metrics/BlockLength
```

Comment:
I actually like this Cop, I think it encourages better design by building objects that follow SRP. Here I was being lazy, I think a better solution may be some kind of `column` method that adds columns to the model (builds the column_names array behind the scenes), and if we got carried away we could add type information and use that when building SQL statements to avoid rails wrapping numbers in quotes.

That being said, I'm not opposed to disabling it globally... I would be tempted to maintain another configuration that I could run locally on new code that I work on to enforce this.